docs/release-playbook.md

```diff
@@ -0,0 +1,205 @@
+    cherry-pick (after the initial candidate), N will be 2.
+
+    ```
+    scripts/release/release.sh create --force_rc=2 $RELEASE_NUMBER $BASELINE_COMMIT [CHERRY_PICKS...]
+    ```
+
+*   If you already did some cherry-picks and you want to add more, use "git log"
+    to find the latest commit (this corresponds to the last cherry-pick commit).
+    Use that as the new baseline and list the new cherry-picks to add on top. Or
+    simply re-use the same baseline and cherrypicks from the previous candidate,
+    and add the new cherrypicks.
+
+    ```
+    scripts/release/release.sh create --force_rc=3 $RELEASE_NUMBER NEW_BASELINE_COMMIT [NEW_CHERRY_PICKS...]
+    ```
+
+*   Resolve conflicts if there are any, type `exit` when you are done, then the script will continue.
+    *   WARNING: `release.sh create` handles conflicts in a subshell (which is why you need to type `exit`).
+
+*   Check/edit release notes.
+
+*   Run `release.sh push`. This uploads the candidate and starts the release
+    process on BuildKite.
+
+    ```
+    scripts/release/release.sh push
+    ```
+
+*   Update GitHub issue with the command that was run and the new candidate name
+    (ie, 0.19.1rc3).
+
+*   Check BuildKite results at https://buildkite.com/bazel/release. You should
+    see the `release-$RELEASE_NUMBER` branch here and a new build running for
+    your release.
+
+    *   If the build fails with "Build creator not allowed", simply start a new
+        one by clicking on the "New build" button in the top right corner
+        ([Issue
+        #281](https://github.com/bazelbuild/continuous-integration/issues/281)).
+
+*   Check the postsubmit test run for the release branch to ensure that all
+    tests on all platforms pass with the version you're about to release.
+
+    *   Go to https://buildkite.com/bazel/bazel-bazel and find the
+        `release-$RELEASE_NUMBER` branch in the list. A build should
+        automatically run. Make sure that it passes.
+
+*   When it all looks good, go back to the job in the release pipeline, click
+    "Unblock step" for the deployment step. This will upload the release
+    candidate binaries to GitHub, https://releases.bazel.build and our apt-get
+    repository.
+
+    *   If you don't have the permission, ask one of the Buildkite org
+        admins to add you to the
+        [bazel-sheriffs](https://buildkite.com/bazel?team=bazel-sheriffs)
+        group.
+
+    *   If that worked, click "Unblock step" for the "Generate Announcement" step.
+
+*   Prepare the release announcement on http://go/bazel-newsletters.
+    *   Create a new section for the release. Populate using the generated text
+        (from the "generate announcement" step).
+    *   Reorganize the notes per category (C++, Java, etc.)
+    *   Add a comment with "+[spomorski@google.com](mailto:spomorski@google.com)" so that he takes a look.
+    *   Send an email to [blaze-devel](mailto:blaze-devel@google.com) asking for reviewers.
```

Comment:
Remove. We could send an email to everyone who has added any release note.

For now, you can just replace `blaze-devel` with `bazel-dev` (https://groups.google.com/forum/#!forum/bazel-dev)

Reply:
Done.